Problem in serializing ImmutableRatesProvider

Serializing ImmutableRatesProvider through JodaBeanSer returns an error (see below for the code which generated the error).

The error message is:
FAILED: 
java.lang.IllegalArgumentException: Unable to convert type com.opengamma.strata.basics.currency.Currency declared as com.opengamma.strata.basics.currency.Currency
	at org.joda.beans.ser.xml.JodaBeanXmlWriter.writeSimple(JodaBeanXmlWriter.java:379)
	at org.joda.beans.ser.xml.JodaBeanXmlWriter.writeBean(JodaBeanXmlWriter.java:198)
	at org.joda.beans.ser.xml.JodaBeanXmlWriter.writeBean(JodaBeanXmlWriter.java:219)
	at org.joda.beans.ser.xml.JodaBeanXmlWriter.writeBean(JodaBeanXmlWriter.java:191)
	at org.joda.beans.ser.xml.JodaBeanXmlWriter.writeToBuilder(JodaBeanXmlWriter.java:168)
	at org.joda.beans.ser.xml.JodaBeanXmlWriter.write(JodaBeanXmlWriter.java:132)
	at org.joda.beans.ser.xml.JodaBeanXmlWriter.write(JodaBeanXmlWriter.java:121)
	at com.opengamma.strata.pricer.curve.CalibrationDiscountingSimple1Test.calibration_present_value(CalibrationDiscountingSimple1Test.java:191)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
Caused by: java.lang.IllegalArgumentException: object is not an instance of declaring class
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.joda.convert.ReflectionStringConverter.convertToString(ReflectionStringConverter.java:64)
	at org.joda.convert.StringConvert.convertToString(StringConvert.java:290)
	at org.joda.beans.ser.xml.JodaBeanXmlWriter.writeSimple(JodaBeanXmlWriter.java:371)

The following code was introduced in 'CalibrationDiscountingSimple1Test.calibration_present_value' to produce the error:

ImmutableRatesProvider multicurve =
        CALIBRATOR.calibrate(CURVE_GROUP_DEFN, ALL_QUOTES, REF_DATA);

JodaBeanSer.PRETTY.xmlWriter().write(multicurve);

--- a/modules/pom.xml
+++ b/modules/pom.xml
@@ -517,7 +517,7 @@
     <findbugs.version>1.3.9</findbugs.version>
     <guava.version>19.0</guava.version>
     <joda-convert.version>1.8.1</joda-convert.version>
-    <joda-beans.version>1.7</joda-beans.version>
+    <joda-beans.version>1.8</joda-beans.version>
     <slf4j.version>1.7.21</slf4j.version>
     <jcommander.version>1.48</jcommander.version>
     <testng.version>6.9.10</testng.version>

